From 09128b9e09b21ea3b4a7e5e19b211fc5f4c8c564 Mon Sep 17 00:00:00 2001
From: NeilBrown <neilb@suse.de>
Date: Tue, 7 Jun 2022 15:59:14 -0400
Subject: [PATCH] autoconf: change tirpc to check for a file, not for an
 include

Recent autoconf don't like variables in AC_CHECK_INCLUDE so we get a
warning.
In libtirpc.m4 we only need to check for the existence of a file, we
don't need to extra  support for includes, such as defining HAVE_TIRPC_H
or whatever.

So change from AC_CHECK_INCLUDE to AC_CHECK_FILE.

Signed-off-by: NeilBrown <neilb@suse.de>
Signed-off-by: Steve Dickson <steved@redhat.com>
---
 aclocal/libtirpc.m4 | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/aclocal/libtirpc.m4 b/aclocal/libtirpc.m4
index f7de5193c177..bddae0226931 100644
--- a/aclocal/libtirpc.m4
+++ b/aclocal/libtirpc.m4
@@ -49,9 +49,9 @@ AC_DEFUN([AC_LIBTIRPC_OLD], [
   dnl Also must have the headers installed where we expect
   dnl to look for headers; add -I compiler option if found
   AS_IF([test "$has_libtirpc" = "yes"],
-        [AC_CHECK_HEADERS([${tirpc_header_dir}/netconfig.h],
-                          [AC_SUBST([AM_CPPFLAGS], ["-I${tirpc_header_dir}"])],
-                          [has_libtirpc="no"])])
+        [AC_CHECK_FILE([${tirpc_header_dir}/netconfig.h],
+                       [AC_SUBST([AM_CPPFLAGS], ["-I${tirpc_header_dir}"])],
+                       [has_libtirpc="no"])])
 
   dnl Now set $LIBTIRPC accordingly
   AS_IF([test "$has_libtirpc" = "yes"],
-- 
2.36.1

